code/style.css

```diff
@@ -1,4 +1,334 @@
-html {
-  background: #0025ff;
-  color: #fff;
+body {
+  margin: 0;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+/* Header */
+
+h1 {
+  margin: 0;
+  font-family: Georgia, "Times New Roman", Times, serif;
+}
+
+nav {
+  background-color: white;
+  color: black;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  border-bottom: solid;
+  border-color: cornflowerblue;
+}
+
+.leftlinks {
+  display: none;
+  text-decoration: none;
+  color: black;
+  font-weight: bold;
+  margin-left: 30px;
+  margin-right: 30px;
+}
+
+.rightlinks {
+  display: none;
+  text-decoration: none;
+  color: black;
+  font-weight: bold;
+  margin-left: 30px;
+  margin-right: 30px;
+}
+.rightlinks:hover {
+  color: blue;
+}
+
+.leftlinks:hover {
+  color: blue;
+}
+
+.logo {
+  height: 50px;
+  margin: 10px 20px;
+}
+nav {
+  display: flex;
+}
+
+nav a {
+  color: #575555;
+  padding: 23px 30px;
+  text-decoration: none;
+  font-size: 14px;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  letter-spacing: 3px;
+  align-self: flex;
+}
+
+nav a:hover {
+  color: grey;
+  background-color: lightgrey;
+}
+
+/* BIG NEWS BANNER */
+
+.big-news-container {
+  margin: 40px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.big-news-container img {
+  object-fit: contain;
+  height: 350px;
+}
+
+.readmore {
+  font-weight: bold;
+  font-style: italic;
+  background-color: cyan;
+  color: black;
+  text-decoration: none;
+  margin-top: 20px;
+  padding: 10px;
+  box-shadow: 4px 5px black;
+}
+
+.readmore:hover {
+  background-color: rgb(175, 32, 32);
+  color: black;
+}
+
+hr {
+  height: 5px;
+  border: none;
+  background-color: black;
+  width: 90%;
+}
+
+.latestnews {
+  margin-left: 50px;
+  margin-bottom: 0;
+  font-weight: bold;
+  font-size: 30px;
+  font-style: italic;
+  color: black;
+  text-shadow: 2px 2px rgb(73, 73, 73);
+}
+
+hr {
+  border: 0;
+  height: 1px;
+  background-image: linear-gradient(to right, blue, grey, blue);
+}
+
+footer {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 0px;
+  background-color: #f6f4e6;
+  width: 100%;
+  height: 125px;
+}
+
+/* NEWS CARDS SECTION STYLING*/
+
+.news-cards-container {
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  align-items: stretch;
+  align-content: center;
+}
+
+.cards {
+  background-color: #e5e5e5;
+  display: flex;
+  flex-direction: column;
+  width: 90%;
+  flex-basis: auto;
+  margin: 20px 0;
+  position: relative;
+  border: 5px solid transparent;
+  text-align: center;
+}
+
+.cards img {
+  align-self: center;
+  width: 100%;
+  height: 450px;
+  margin: 20px;
+}
+
+h3 {
+  padding-left: 10px;
+  padding-right: 10px;
+  margin-bottom: 70px;
+}
+
+/* MEDIA QUERIES */
+
+/* Mobile */
+
+@media (max-width: 666) {
+  .leftlinks {
+    display: none;
+  }
+
+  .rightlinks {
+    display: none;
+  }
+}
```

Comment:
If you code mobile first, you don't need a media query for mobile since it's default :)